### DISCUSSION TITLE
Raspberry pi3

### DISCUSSION THREAD

**Discussion by nemessmu** [PiotrMachowski/Xiaomi-cloud-tokens-extractor#28] [Q&A] [answered]
How can I install this on Ha ? docker run --rm -it $(docker build -q .) this, or pip3 install requests , not working for me at all :(
Can someone help me out, im still a newbie

**Comment by PiotrMachowski** [comment]
You don't have to do it on HA machine, it can be done on any device, even outside of your network :)

**Comment by PiotrMachowski** [comment, marked as answer]
I have added an [instruction for HA OS](https://github.com/PiotrMachowski/Xiaomi-cloud-tokens-extractor#home-assistant---ssh--web-terminal)
